docs/unified-test-documentation/dasharo-compatibility/359-battery-boot-block.md

```diff
@@ -0,0 +1,81 @@
+# Dasharo Compatibility: Battery boot block
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup#os-boot-from-disk).
+1. DUT not plugged in to the charger.
+
+## BBB001.001 Boot block when battery level is low (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify that booting into the OS is blocked when the battery
+level sits below 5%.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+    [Test cases common documentation](#test-cases-common-documentation) section.
+
+**Test steps**
+
+1. Power on the DUT.
+1. Boot into the system.
+1. Log into the system by using the proper login and password.
+1. Open a terminal window and install `stress-ng` package by executing following
+    command:
+
+```bash
+sudo apt install stress-ng
```

Comment:
We can do it without installing any additional packages, see: https://superuser.com/questions/443406/how-can-i-produce-high-cpu-load-on-a-linux-server

Second answer gives some nice shell one-liners and a nice function that also allows specifying a timeout.

Reply:
I used it because it's easily configurable, for example I can set a specific timeout of the stress test after which the battery level is checked and so on. We also use the package in other test cases. Why not use software specifically made for our purpose when it's available?

Reply:
THe methods mentioned there also allow specifying timeout